pkgs/development/libraries/audio/libgme/default.nix

```diff
@@ -21,4 +21,12 @@ in stdenv.mkDerivation {
   };
 
   buildInputs = [ cmake ];
+
+  # It used to reference it, in the past, but thanks to the postFixup hook, now
+  # it doesn't
+  disallowedReferences = [ gcc-unwrapped ];
+
+  postFixup = ''
+    sed -i 's,${gcc-unwrapped},/gcc,g' "$(readlink -f $out/lib/libgme.so)"
+  '';
```

Comment:
maybe removeReferencesTo would be more adapted here. Using sed on binary files looks strange.

Reply:
Yea I tried this:

```nix
    ${removeReferencesTo}/bin/remove-references-to ${gcc-unwrapped} "$(readlink $out/lib/libgme.so)"
```

but it just didn't work... It also didn't work with it in the `nativeBuildInputs`.

Reply:
`remove-references-to` expects to be passed `-t` option to specify what reference(s) should be removed. Tho it does use `sed` itself under the hood to clear out the references so... :-D. In any case I'm more worried about replacing symbols in a binary file / object with something that does not match the exact same length, I would be surprised if that didn't corrupt it (something `remove-references-to` takes care of for you).

Reply:
Yes, this is extremely likely to break the library as is.

Reply:
> remove-references-to expects to be passed -t option

OK great (it's not documented).

> Yes, this is extremely likely to break the library as is.

OK, good to know. Thanks!